eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -427,6 +427,15 @@ abstract class SafeMojo extends AbstractMojo {
     @SuppressWarnings("PMD.ImmutableField")
     protected BiConsumer<Dependency, Path> central;
 
+    /**
+     * The Git hash to pull objects from.
+     * If not set, will be computed from {@code tag} field.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    protected CommitHash hash = new CommitHash.ChConstant(
```

Comment:
@vl4ds4m let's use `ChCached` (caching decorator) instead of `ChConst`. This will allow not to call `.value()` explicitly on the next line, in the same way it's implemented in `ProbeMojo`